test/fulfillment_order_test.rb

```diff
@@ -0,0 +1,218 @@
+require 'test_helper'
+
+class FulFillmentOrderTest < Test::Unit::TestCase
+  def setup
+    super
+    fake "orders/450789469/fulfillment_orders/519788021", method: :get,
+      body: load_fixture('fulfillment_order')
+  end
+
+  context "FulfillmentOrder" do
+    context "#find" do
+      should "be able to find fulfillment order" do
+        fulfillment_order = ShopifyAPI::FulfillmentOrder.find(519788021, params: { order_id: 450789469 })
+        assert_equal 519788021, fulfillment_order.id
+        assert_equal 450789469, fulfillment_order.order_id
+      end
+    end
+
+    context "#all" do
+      should "be able to list fulfillment orders for an order" do
+        fake 'orders/450789469/fulfillment_orders', method: :get, body: "[#{load_fixture('fulfillment_order')}]"
+
+        fulfillment_orders = ShopifyAPI::FulfillmentOrder.all(
+          params: { order_id: 450789469 }
+        )
+
+        assert_equal 1, fulfillment_orders.count
+        fulfillment_order = fulfillment_orders.first
+        assert_equal 519788021, fulfillment_order.id
+        assert_equal 450789469, fulfillment_order.order_id
+      end
+    end
+
+    # context "#with_assigned_status" do
```

Comment:
what is up with this commented out code?

Reply:
I was experimenting with how to organize assigned_fulfillment_orders either as a resource type matching the implemented endpoints or as methods on a fulfillment_orders resource from the client's perspective.

This can be deleted put into an assigned_fulfillment_orders_test.